tbump/test/test_cli.py

```diff
@@ -118,15 +119,27 @@ def only_patch_done(test_repo: Path, previous_commit: str) -> bool:
     )
 
 
+@pytest.fixture
+def message_recorder() -> Iterator[MessageRecorder]:
```

Comment:
there's a cli_ui fixture you can use to test calls to `ui.info()`

Reply:
Ah, much nicer. :smile: